cypress/pageObjects/signIn/assertions.ts

```diff
@@ -5,6 +5,11 @@ class SignInPageAssertions {
       .should("contain", content);
     return this;
   }
+
+  checkLoginURL() {
```

Comment:
after merging this PR https://github.com/AzizaAlSayed/cypress-test-repo/pull/12
you can use the checkUrlContainsValue assertion.